src/common/cockpitpipe.c

```diff
@@ -440,6 +441,9 @@ set_problem_from_errno (CockpitPipe *self,
     problem = "access-denied";
   else if (errn == ENOENT || errn == ECONNREFUSED)
     problem = "not-found";
+  /* only warn about Cockpit-internal fds, not opaque user ones */
+  else if (errn == EBADF && self->priv->is_user_fd)
+    problem = "internal-error";
```

Comment:
"protocol-error" or "not-supported" are also close, but neither fits really well.